src/Sulu/Bundle/RouteBundle/Resources/config/doctrine/Route.orm.xml

```diff
@@ -6,6 +6,10 @@
     <entity repository-class="Sulu\Bundle\RouteBundle\Entity\RouteRepository"
```

Comment:
this is not configured as a `mapped-superclass` right now. do we need to change this?